app/mail_sender.py

```diff
@@ -23,22 +30,72 @@ class SendRequest:
     is_forward: bool = False
     ignore_smtp_errors: bool = False
 
+    def to_bytes(self) -> bytes:
+        if not config.SAVE_UNSENT_DIR:
+            return
+        if not os.path.isdir(config.SAVE_UNSENT_DIR):
```

Comment:
I wonder if  we can do this directory creation in `config` instead? Similarly to https://github.com/simple-login/app/blob/15810c2382cc81e6312e7a8484cc47af1a278a78/app/config.py#L226

Reply:
Done